src/notation/view/pianokeyboard/pianokeyboardpanelcontextmenumodel.h

```diff
@@ -43,6 +43,7 @@ class PianoKeyboardPanelContextMenuModel : public muse::uicomponents::AbstractMe
 
     Q_PROPERTY(int numberOfKeys READ numberOfKeys NOTIFY numberOfKeysChanged)
     Q_PROPERTY(qreal keyWidthScaling READ keyWidthScaling WRITE setKeyWidthScaling NOTIFY keyWidthScalingChanged)
+    Q_PROPERTY(bool pianoKeyboardNotatedPitch READ pianoKeyboardNotatedPitch NOTIFY pianoKeyboardNotatedPitchChanged)
```

Comment:
This Q_PROPERTY is (currently) not used in QML, so there is no need to add it. It looks like the `pianoKeyboardNotatedPitchChanged` signal is not used either, so that can be removed too. Maybe even the `pianoKeyboardNotatedPitch` function too, but perhaps that one _is_ useful.

Reply:
We ended up deciding on using the signal but I'm not sure if that's the best way of doing what we are trying to do.